### PR TITLE
rework workflows

### PR DESCRIPTION
**Pull request by lilyanB**

### CODE REVIEW

File: .github/workflows/age-ci.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Age CI
 on:
   push:
     paths:
-      - age/**
+      - age/contracts/**
```
Comment (gregLibert) [inline .github/workflows/age-ci.yml:6]: Why ? Don't we want to rerun it if the front change ?

Also true for the others.

File: .github/workflows/age-daily-deploy.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Age Daily Deploy
 on:
   push:
     paths:
-      - age/**
+      - age/contracts/**
```
Comment (gregLibert) [inline .github/workflows/age-daily-deploy.yml:6]: Why is it here if it's only the daily part ?

Also true for the others.

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: .github/workflows/age-ci.yml (hunk 1 above)
Comment (lilyanB) [reply .github/workflows/age-ci.yml:6]: Because these tests start with the contract file

Comment (lilyanB) [reply .github/workflows/age-ci.yml:6]: But, I follow your instruction if you prefer to do it

File: .github/workflows/age-daily-deploy.yml (hunk 2 above)
Comment (lilyanB) [reply .github/workflows/age-daily-deploy.yml:6]: What do you mean? I didn't understand

File: .github/workflows/age-ci.yml (hunk 1 above)
Comment (lilyanB) [reply .github/workflows/age-ci.yml:6]: done

File: .github/workflows/age-daily-deploy.yml (hunk 2 above)
Comment (lilyanB) [reply .github/workflows/age-daily-deploy.yml]: done

**Comment by Ben-Rey** [comment]
@gregLibert Can we merge this one, it seems to fix the ci

**Review by gregLibert — APPROVED** [review]

**Pull request merged**